doc/modules/model_evaluation.rst

```diff
@@ -246,12 +246,12 @@ permitted and will require a wrapper to return a single metric::
     >>> # A sample toy binary classification dataset
     >>> X, y = datasets.make_classification(n_classes=2, random_state=0)
     >>> svm = LinearSVC(random_state=0)
-    >>> def tn(y_true, y_pred): return confusion_matrix(y_true, y_pred)[0, 0]
-    >>> def fp(y_true, y_pred): return confusion_matrix(y_true, y_pred)[0, 1]
-    >>> def fn(y_true, y_pred): return confusion_matrix(y_true, y_pred)[1, 0]
-    >>> def tp(y_true, y_pred): return confusion_matrix(y_true, y_pred)[1, 1]
-    >>> scoring = {'tp' : make_scorer(tp), 'tn' : make_scorer(tn),
-    ...            'fp' : make_scorer(fp), 'fn' : make_scorer(fn)}
+    >>> tn = lambda y_true, y_pred: confusion_matrix(y_true, y_pred)[0, 0]
```

Comment:
This is also not acceptable in PEP8. Needs to be two-line functions with one blank line between each. I'd rather what's already there.

Reply:
You are right, but in this case it seems to me better to declare a one-line lambda than a function.
There is some set of Pep8 exceptions, for example, indents after imports, classes and functions. In order to make this code look like the one they write in the console.

Reply:
I'll vote +1 for the original version because they're much more straightforward to many users.

Reply:
Sure no problem

Reply:
Reverted